src/it/projects/complete-multimodule-parent-pom-cifriendly/pom.xml

```diff
@@ -97,7 +97,7 @@
 
 	<properties>
 		<!-- Default revision -->
-		<revision>1.2.3.0</revision>
+		<revision>1.2.3.4</revision>
```

Comment:
Could you give a rationale for this change. Its been a while but wasnt the purpose to test the overriding of this default revision from maven.config to `1.2.3.4` so with this change the beviour could break and the test would not reveal that anymore?

Reply:
My fault, it was a distraction. I did that change because I wanted to test other stuff, now I changed it back as it was before